### DISCUSSION TITLE
Does MOOSE have 'drainage-only flow' BCs like Abaqus

### DISCUSSION THREAD

**Discussion by Traiwit** [idaholab/moose#18198] [Q&A Modules: General] [answered]
Hi guys,

I'm wondering if MOOSE has 'drainage-only flow BCs similar to Abaqus? 

[LINK](https://abaqus-docs.mit.edu/2017/English/SIMACAEPRCRefMap/simaprc-c-porousflow.htm
)

where we control fluid_vel at the boundary instead of specifying pore pressure at the boundary (Dirichlet BCs)

![1](https://user-images.githubusercontent.com/79562439/123736305-3ba8e680-d8e4-11eb-80c7-0e8bcec28f5f.PNG)

Kind regards,
Traiwit

**Comment by WilkAndy** [comment, marked as answer]
Yes, look at https://mooseframework.inl.gov/modules/porous_flow/boundaries.html and https://mooseframework.inl.gov/source/bcs/NeumannBC.html

**Reply by Traiwit** [reply]
Awesome @WilkAndy 

if I understand correctly, this is what I am looking for, right?

![2](https://user-images.githubusercontent.com/79562439/123737702-c7bc0d80-d8e6-11eb-9b20-7b409679bd70.PNG)

![3](https://user-images.githubusercontent.com/79562439/123737750-defafb00-d8e6-11eb-818c-d14e900c4d1e.PNG)

**Reply by WilkAndy** [reply]
you got it!   You can also utilise `use_*` things to make the slope of that graph a meaningful number (also documented on that page)

**Reply by Traiwit** [reply]
awesome, thank you!
